### ISSUE TITLE
Richer data in api/matches...

### ISSUE THREAD

**Issue by vishalinp** [datdota/datdota#20]
Hi,

Firstly, thank you for the amazingly well curated Dota 2 match data. On to my request:

Would you consider adding richer data to the datdota.com/matches/[matchid] API call? The data contained in the API call is quite sparse compared to what can be seen on the actual match page. The page itself contains **Heroes**, **Players**, **Levels**, **Kills**, **Deaths** etc. for both teams. It would be great if all of this could be included in the json returned.

**Comment by Noxville** [comment]
Hey there. Yeah, this has been something I've wanted to improve for a long while. Do you think just having parity with the match page is fine; or is there extra information you think is required?

**Comment by vishalinp** [comment]
Parity with match page would be great, since this has most of the essentials already. If you're looking for a second opinion on this though, then I can recommend including ending items, and end status of towers/structures. Picks and bans would be the final bit of the essentials. 

Any other sort of interesting data is likely to be intra-match (time stamped), and so probably not suited to this specific request.

**Comment by Noxville** [comment]
I made a first stab at this: http://datdota.com/api/matches/4562402022 - let me know how it looks to you

I'll make some adjustments and add the draft stuff later if I get a chance.

**Comment by vishalinp** [comment]
Looks great! Time-stamped item purchases and skill choices was a good addition. I'll let you know if I think of anything else.

**Comment by Noxville** [comment]
Yeah, I plan on doing a full /match/$matchId visual rework - so as more stuff is required for that page, more data will have to be included in the API (the frontend basically renders using the data from t he backend).

**Comment by Noxville** [comment]
Just a self-reminder: draft data is the next bit coming here.
